configurable colors for printed todo list

bdzimmer opened the pull request:
I wanted to be able to change the color that the filename is printed with, so I added an object to the settings that holds colors to use for the filename, line number, and line.

I don't know what the best scope for the Colors class is. It would be nice if it was easier to change in `build.sbt`. As is, I think you have to do something like this:

`todosColors := com.github.fedragon.TodoList.Colors(scala.Console.RED, scala.Console.GREEN, scala.Console.BLUE)`

Thanks for writing this plugin! It's very useful, and by looking at it I learned a lot about sbt and sbt plugins.

bdzimmer committed:
configurable colors for printed todo list

fedragon commented:
Hi @bdzimmer, thank you very much for this pull request!

I added a few comments to your changes, please let me know what you think about them!

bdzimmer committed:
individual setting keys for colors

bdzimmer commented:
Rebased and added individual keys for the different colors. Added a default Colors object to TodoList. I suppose the Colors class isn't necessary, but I like it since it indicates clearly which colors are required (as well as stores defaults) if more need to be added later.

fedragon commented:
merged! thank you very much for your contribution, I will release a new version later this morning!
